src/whylogs/core/summaryconverters.py

```diff
@@ -118,6 +121,41 @@ def single_quantile_from_sketch(sketch: kll_floats_sketch, quantile: float):
     return type("Object", (), {"quantile": qval[0]})
 
 
+def _calculate_bins(end: float, start: float, n: int, avg_per_bucket: float, max_buckets: int):
+    # Include the max value in the right-most bin
+    end += abs(end) * 1e-7
```

Comment:
This will still be 0 if end is passed in as 0... don't want to take log of 0. Probably OK if start != end, but we should ensure we don't take log of 0.

Reply:
I don't think you can get this to call log 0, because we don't log2(end), we log2((end-start)/buckets), and we check for start == end already. I added some new tests to cover when start and end are close to each other and close to 0, also having end == 0.